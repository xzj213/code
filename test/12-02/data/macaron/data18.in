14
841 841 841 841 0 0 0 841 0 841 0 0 0 0
841 841 841 0 0 0 0 841 0 841 841 841 0 841
841 841 0 0 841 0 861 841 841 841 841 0 0 0
841 0 841 841 0 841 841 0 841 841 841 0 841 0
841 0 0 841 841 841 841 841 0 213 0 0 841 0
841 0 0 841 0 0 0 841 841 0 841 841 841 841
0 841 841 841 0 0 0 0 841 0 841 0 0 0
841 0 841 841 841 0 841 0 0 0 0 841 841 841
841 841 841 0 841 0 0 841 841 841 0 0 0 0
0 0 841 841 0 0 0 0 841 841 0 841 0 841
0 841 841 0 0 0 841 0 841 0 841 841 0 841
0 0 841 841 841 0 0 0 0 0 841 841 0 0
0 841 841 0 841 841 841 0 841 841 841 0 841 841
0 841 841 0 841 0 841 841 841 841 841 0 0 841
