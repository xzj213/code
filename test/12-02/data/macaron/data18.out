0 20 193 213 841 861 904 924 
